types/powerapps-component-framework/componentframework.d.ts

```diff
@@ -1640,7 +2357,7 @@ declare namespace ComponentFramework {
 
         /**
          * Entity metadata refer to online documentation
-         * https://learn.microsoft.com/power-apps/developer/model-driven-apps/clientapi/reference/xrm-utility/getentitymetadata
```

Comment:
I think the old doc link is more correct - the new one goes to `customer-engagement` which I believe is for OnPrem. So I'd revet this line

Reply:
There are two other link updates I'd revert

Reply:
Reverted.